src/Controller/ROIOptionsController.py

```diff
@@ -3,39 +3,47 @@
 from src.Controller.AddOnOptionsController import *
 from pydicom import Dataset
 
+
 # Create the ROI Delete Options class based on the UI from the file in View/ROI Delete Option
 class RoiDeleteOptions(QtWidgets.QMainWindow, UIDeleteROIWindow):
     deleting_rois_structure_tuple = QtCore.pyqtSignal(tuple)  # new PyDicom dataset
-    def __init__(self, window, rois, dataset_rtss):
+
+    def __init__(self, rois, dataset_rtss):
         super(RoiDeleteOptions, self).__init__()
 
-        self.window = window
         self.setup_ui(self, rois, dataset_rtss, self.deleting_rois_structure_tuple)
+
+
 # The class that will be called by the main page to access the ROI Options controller
```

Comment:
This is a useful comment, but it should be done as a docstring.
vscode has plugins to auto-populate docstrings, perhaps PyCharm has something similar.